javascript.js

```diff
@@ -1,50 +1,77 @@
-let library = [];
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
 
-function getInput() {
-  const book = {};
-  book.id = Date.now();
-  book.title = document.getElementById('bookTitle').value;
-  book.author = document.getElementById('bookAuthor').value;
-  return book;
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = Math.random();
+  }
+}
+
+class Library {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    this.data.push(book);
+    localStorage.setItem('library', JSON.stringify(this.data));
+    addToUI(book);
+  }
```

Comment:
- [ ] Optional - 
I suggest you favour the use of array methods that don't mutate the array like `concat`, `filter`, `slice` as you did in the removeBooks function